assignment_module06/module06_student01_TranDacNhatAnh/module06_assignment01_student01_TranDacNhatAnh.py

```diff
@@ -0,0 +1,83 @@
+# Mục tiêu của bài tập là so sánh tốc độ khi sử dụng các phép toán tính tổng khi sử dụng thư viện Numpy với khi sử dụng các cách 
+# cài đặt thủ công.
+
+import random as rd
+import numpy as np
+import time
+import csv
+
+
+testResults = []
+for test in range(100):
+    # (a)
+    # Tạo một mảng 2 chiều gồm m hàng và n cột, các hàm tính tổng như đề bài yêu cầu sẽ tính toán sử dụng mảng này.
+    def generateArray(m: int, n: int):
+        someArray = [[rd.randint(0, 10000) for _ in range(n)] for _ in range(m)]
+        return someArray
+
+    startTime1 = time.perf_counter()
+    arr1 = generateArray(3, 4)
+    endTime1 = time.perf_counter()
+    generateArrayTime = endTime1 - startTime1
+
+    # (b)
+    # Tính tổng các số thuộc cùng một cột (được đánh số 0, 1, ..., n-1)
+    def sumInAColumnOfArray(array: list[list[int]]):
+        nRow = len(array)
+        nCol = len(array[0])
+        total = [0 for _ in range(nCol)]
+        for row in range(nRow):
+            for col in range(nCol):
+                total[col] += array[row][col]
+        return total
+
+    startTime2 = time.perf_counter()
+    total1 = sumInAColumnOfArray(array=arr1)
+    endTime2 = time.perf_counter()
+    calculateTotal1Time = endTime2 - startTime2
```

Comment:
Dòng 34-37 này Na ko dùng ở dưới nữa nhỉ

Reply:
thra nó vẫn dùng (để tạo arr2, vì np.ndarrray có thể truy cập như là list[list] nhưng ngược lại thì không, còn t muốn dùng cùng 1 mảng để tính toán :v)